File: Laboratorio hackeramk/02 JavaScript/03_encapsulamiento.js
class Cuenta {
    #saldo;

    constructor(saldo) {
        this.#saldo = saldo;
    }

    depositar(cantidad) {
        this.#saldo += cantidad;
    }

    mostrarSaldo() {
        console.log("Saldo:", this.#saldo);
    }
}

const cuenta = new Cuenta(100);
cuenta.depositar(50);
cuenta.mostrarSaldo();

File: Laboratorio hackeramk/03 C Sharp/03_encapsulamiento.cs
using System;

class Cuenta
{
    private double saldo;

    public Cuenta(double saldo)
    {
        this.saldo = saldo;
    }

    public void Depositar(double cantidad)
    {
        saldo += cantidad;
    }

    public void MostrarSaldo()
    {
        Console.WriteLine("Saldo: " + saldo);
    }

    static void Main()
    {
        Cuenta cuenta = new Cuenta(100);
        cuenta.Depositar(50);
        cuenta.MostrarSaldo();
    }
}
